hasher-matcher-actioner/webapp/src/settings/ActionSettingsTab.jsx

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+/* eslint-disable react/prop-types */
+
+import React, {useState} from 'react';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Table from 'react-bootstrap/Table';
+import Form from 'react-bootstrap/Form';
+
+export default function ActionSettingsTab() {
+  return (
+    <>
+      <ActionPerformerSettings />
+      <ActionLabelSettings />
+    </>
+  );
+}
+
+function ActionLabelSettings() {
+  return <>Action Label Settings coming here</>;
+}
+
+const ActionerTypes = {
+  WebhookActioner: {
+    args: {
+      url: {description: 'The url to send a webhook to'},
+      webhookType: {
+        description: 'What type of webhook should be sent?',
+        default: 'POST',
+      },
+      headers: {
+        description: 'Optional json object of headers to include in webhook',
+        default: '{}',
+      },
+    },
+    description:
+      'When a match occurs, a webhook will be sent to the specified url with data describing the match',
+  },
+};
+
+function WebhookActionerRender({url, webhookType, headers}) {
+  const actionerDetails = ActionerTypes.WebhookActioner;
+
+  const [editing, setEditing] = useState(false);
+
+  const [savedURL, setURL] = useState(url);
+  const [savedWebhookType, setWebhookType] = useState(webhookType);
+  const [savedHeaders, setHeaders] = useState(headers);
+  const [tempURL, setTempURL] = useState(url);
+  const [tempWebhookType, setTempWebhookType] = useState(webhookType);
+  const [tempHeaders, setTempHeaders] = useState(headers);
+
+  return (
+    <Card>
+      <div hidden={editing}>
+        <Card.Header>
+          WebhookActioner
+          <br />
+          <Form.Text className="text-muted">
+            {ActionerTypes.WebhookActioner.description}
+          </Form.Text>
+        </Card.Header>
+        <Card.Body>
+          URL : {savedURL}
+          <br />
+          Webhook Type : {savedWebhookType}
+          <br />
+          Headers : {savedHeaders}
+          <br />
+        </Card.Body>
+        <Card.Footer>
+          <Button
+            variant="primary"
+            onClick={() => {
+              setEditing(true);
+            }}>
+            Edit
+          </Button>
+        </Card.Footer>
+      </div>
+
+      <div hidden={!editing}>
+        <Card.Header>
+          <Form>
```

Comment:
I'm not 100% sure, but when you get to wiring this up, you do have two forms here, one for the action "type" and one for all of the details... you may or may not find it necessary to have one outer <Form>... just something to be on the lookout for. It might be find to put <Form> just outside the card header, body and footer.

Reply:
Might have to reorganize some of this when we get to multiple actioners anyway